x86_build.bat

```diff
@@ -0,0 +1,3 @@
+Add MD5 to the build etc...
```

Comment:
[Weak/Broken Hash Algorithm](https://github.com/microsoft/devskim/guidance/DS126858.md)
A weak or broken hash algorithm was detected.Replacement: Consider switching to use SHA-256 or SHA-512 instead.


Suggestions:
**[1]** *+Add SHA256 to the build etc...*
**[2]** *+Add SHA512 to the build etc...*